stm32_loader.py

```diff
@@ -272,7 +284,10 @@ def load(port,
                 loader.synchronize(skip_prefix=True)
 
         # General info
-        logger.info('Target info: %s', loader.get())
+        var_get = loader.get()
+
+        logger.info('Target commands: %s', map(lambda x: '0x%.2x ' % x,
+            var_get['commands']))
```

Comment:
Formatting (should be one line, or the continuation line should be properly indented).